app/javascript/mastodon/utils/time.tsx

```diff
@@ -0,0 +1,29 @@
+import type { ComponentProps } from 'react';
+
+import { FormattedDate } from 'react-intl';
+
+export const FormattedDateWrapper = (
```

Comment:
i'd prefer to have this be in `app/javascript/mastodon/components/formatted_date.tsx` as it's more a component than a utility, and it seems to mostly deal with dates instead of time.

Reply:
moved paths and force-pushed.